examples/solvers/pdhg_denoising_ROF_algorithm_comparison.py

```diff
@@ -0,0 +1,124 @@
+"""Total variation denoising using the primal-dual hybrid gradient algorithm.
```

Comment:
The current version of this file runs 3 algorithms for the ROF (L2^2 + TV) problem. 1) the algorithm of the old example 2) a variant that most people would use for this and 3) the 1/k^2 convergence version that makes use of strong convexity.